app/views/plantings/index.html.haml

```diff
@@ -22,40 +22,71 @@
 
   %table.table.table-striped
     %tr
-      - unless @owner
-        %th Owner
-      %th Crop
-      %th Garden
-      %th Quantity
-      %th Planted on
-      %th Finished
-      %th Sun/shade?
-      %th Planted from
-      %th
-
-    - @plantings.each do |planting|
+      %th #
+      %th Photo
+      %th{:colspan => "3"} Planting details
+    
+    - @plantings.each.with_index do |planting, index|
       %tr
-        - unless @owner
-          %td= link_to planting.owner.login_name, planting.owner
-        %td= link_to planting.crop.name, planting.crop
-        %td= link_to planting.garden.name, planting.garden
-        %td= planting.quantity
-        %td= planting.planted_at
+        %td= "##{index+1}"
+        %td= link_to image_tag((planting.default_photo ?  planting.default_photo.thumbnail_url : 'placeholder_150.png'), :alt => planting.crop_id, :class => 'img'), planting
```

Comment:
I think the code in the body of this loop should be moved into plantings/_thumbnail.html.haml, where it can replace the rather dull planting-summary that we're currently using in Garden detail views. Then replace the loop body here with a call to the thumbnail partial :-)